cmd/snap/cmd_run.go

```diff
@@ -65,6 +68,7 @@ func init() {
 			"hook":    i18n.G("Hook to run"),
 			"r":       i18n.G("Use a specific snap revision when running hook"),
 			"shell":   i18n.G("Run a shell instead of the command (useful for debugging)"),
+			"strace":  i18n.G("Run the command under strace (useful for debugging"),
```

Comment:
Should we mention what's already enabled? `-f -e !select..`

Reply:
A good question, doing this in a concise form will be tricky but it would be worthwhile. Fwiw, we need to exclude these options or strace will hang.